### DISCUSSION TITLE
The ability to change the bot's rich presence

### DISCUSSION THREAD

**Discussion by Zupaaaa** [jagrosh/MusicBot#868] [Questions] [answered]
It would be cool to change what the bot is doing. I'm not the best with code, so it might be easy, I just don't know how to do it. Could anyone help me through it?

**Comment by jagrosh** [comment]
Bots cannot use rich presence

**Comment by jagrosh** [comment, marked as answer]
Check the [config](https://jmusicbot.com/config), you can change the game the bot is "playing"

**Comment by CyberFlameGO** [comment]
There's also (prefix)`setgame [action] [game]`, which sets the game the bot is playing (in memory iirc, so not persistent on bot instance restart)
